### DISCUSSION TITLE
Procedure to move away from Mingw32 to Mingw64

### DISCUSSION THREAD

**Discussion by jmorel33** [raysan5/raylib#1898] [Show and tell]
I've hit a serious barrier with using a 32bit compiler and so I am moving away from that.

I am looking for migrating to Mingw64 and would like to know what is integrated in Mingw32 that makes RayLib stuck on Mingw32 so I can "unstuck" it and properly compile on Mingw64.

Thanks for any tips advice, how-to, etc.

**Comment by raysan5** [comment]
You are free to use MinGW 64bit, afaik, there is nothing in raylib preventing it.

**Reply by jmorel33** [reply]
> You are free to use MinGW 64bit, afaik, there is nothing in raylib preventing it.

I am sure of it, just that with the GCC supplied with RayLib, if you uncomment
```$(RESC) $(NAME_PART).rc -o $(NAME_PART).rc.data --target=pe-x86-64```

It will give an error as follows;
```Process started (PID=11652) >>>
windres: Can't detect target endianness and architecture.
<<< Process finished (PID=11652). (Exit code 1)```

**Reply by raysan5** [reply]
Oh, yeah, the resource file needs to be rebuild for the target architecture.

**Reply by jmorel33** [reply]
Thank you Ray I managed to succeed!  It was really easy, basically just replaced Mingw32 with Mingw64 and replaced target march and few other parameters, paths.  But mostly I used that moment to organize myself better.

Thank you!
